app/views/admin/dashboard/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <h1>Admin Dashboard</h1>
 <div class="uncontacted-companies">
   <% @uncontacted_companies.each do |company| %>
-    <%= company.organization %>
+    <%= link_to company.organization, admin_company_path(company.id) %>
   <% end %>
 </div>
 <div class="contacted-companies">
   <% @contacted_companies.each do |company| %>
-    <%= company.organization %>
+    <%= link_to company.organization, admin_company_path(company.id) %>
```

Comment:
We may need to add two more links for confirmed and dead companies now that my presenter class pr is merged.